src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.stories.js

```diff
@@ -4,6 +4,9 @@ import WebexInterstitialMeeting from './WebexInterstitialMeeting';
 export default {
   title: 'Meetings/Webex Interstitial Meeting',
   component: WebexInterstitialMeeting,
+  parameters: {
+    storyshots: {disable: true},
```

Comment:
are we disabling all storyshots for this component?

Reply:
yes. there is a problem with the modal testing because the snapshot doesn't contain it, that's the reason why I choose to disable the story shots. or we can do it just for one story but this means that none of the other stories would display a meeting witch has the 'asking' value for video permission.

Reply:
I think it's fine to not having any story with "asking".